docs/index.md

```diff
@@ -72,6 +74,10 @@ Gravitino currently supports the following catalogs:
 Gravitino also provides an Iceberg REST catalog service for the Iceberg table format. See the
 [Iceberg REST catalog service](./iceberg-rest-service.md) for details.
```

Comment:
Should we include the message between relational catalog and fileset catalog?
```
Gravitino also provides an Iceberg REST catalog service for the Iceberg table format. See the
[Iceberg REST catalog service](./iceberg-rest-service.md) for details.
```

Reply:
Can you explain more?

Reply:
I think it's more reasonable
```
Gravitino currently supports the following catalogs:
**Relational catalogs:**
* [**Iceberg catalog**](./lakehouse-iceberg-catalog.md)
* [**Hive catalog**](./apache-hive-catalog.md)
* [**MySQL catalog**](./jdbc-mysql-catalog.md)
* [**PostgreSQL catalog**](./jdbc-postgresql-catalog.md)

**Fileset catalogs:**

* [**Hadoop catalog**](./hadoop-catalog.md)

Gravitino also provides an Iceberg REST catalog service for the Iceberg table format. See the
[Iceberg REST catalog service](./iceberg-rest-service.md) for details.

```